app/views/layouts/listnav/_container_link_to.html.haml

```diff
@@ -0,0 +1,43 @@
+- return if object.nil?
+- if object.kind_of?(ActiveRecord::Relation)
+  -# has_many relation
+  - klass = object.klass.base_model
+  - if klass == Container
+    - feature = 'containers'
```

Comment:
> Would it make more sense to just fix the feature name, instead of proliferating this exceptional value further?

@matthewd I absolutely wanted to but it felt out of place in this patch. Also, doing it would be easier after references to it are refactored to just a few places.